src/screens/Settings/components/ChangeHandleDialog.tsx

```diff
@@ -488,6 +491,18 @@ function OwnHandlePage({goToServiceHandle}: {goToServiceHandle: () => void}) {
         </Animated.View>
       )}
       <Animated.View layout={native(LinearTransition)}>
+        {currentAccount?.handle?.includes('bsky.social') && (
```

Comment:
kind of a nit but this should probably be `endsWith`

Reply:
ope yeah let's do that 👍